handler.cpp

```diff
@@ -121,9 +121,15 @@ RequestHandler::Status StaticHandler::HandleRequest(const Request& request, Resp
 
     // save content_type header based on requested file extension
     std::string content_type = get_content_type(abs_path);
-
     // raw byte array
     std::string to_send = read_file(abs_path);
+
+    // handling markdown
+    if (content_type == "markdown") {
+      markdown::Document d;
+      std::string s(d.read(to_send), {});
+    }
+
     std::cout << "Serving file from: " << abs_path << std::endl;
 
     response->SetStatus(Response::OK);
```

Comment:
The Content-type header is set incorrectly since it'll be set to "markdown" however the browser is expecting "text/html"